src/common/scss/form-input.scss

```diff
@@ -69,9 +69,9 @@ textarea {
 
 .error {
   display: flex;
-  align-items: center;
+  align-items: end;
   color: var(--kd-color-text-error);
-  margin-top: 8px;
+  margin-top: 6px;
```

Comment:
What was the reason for this change? It will affect all form inputs.

Reply:
Reverted the changes as they are no longer required